.tekton/operator-bundle-pipeline.yaml

```
@@ -301,83 +301,107 @@ spec:
       value: "$(params.operator-image-repo):$(tasks.determine-operator-image-tag.results.IMAGE_TAG)"
     taskRef:
       name: wait-for-image
+    # This timeout should be kept in sync with the maximum runtime of the PipelineRun for building the operator image.
```

How about?
```suggestion
    # This timeout must be the same as the timeout in `operator-build.yaml`.
```
... and similarly for others.
The difference is that we point to the file name where engineers should look.

---

Commit fd60890042.